assets/scss/page/live.scss

```
@@ -90,6 +90,14 @@ html[data-page='live'] {
   .comment {
     display: block;
     background-color: #393f4f;
+    position: relative;
+    animation: comment_animation 0.75s cubic-bezier(0.6, -0.2, 0.7, 0.05)
+      reverse;
+  }
+
+  .disable_reverse .comment {
+    animation: disable_reverse_comment 0.75s cubic-bezier(0.6, -0.2, 0.7, 0.05)
+      reverse !important;
```

```suggestion
      reverse;
```

`!important` は必要無さそう

---

無くても動かないのは確認してるけどアニメーションを上書きしてるって明示的にしたくてつけてるんだけどキモイなら消す…？

---

あまりむやみに `!important` を使うべきでは無いと思うしまあ多分理解できるだろうから消してしまって良さそう